Resources/public/js/views/services/plugins/ez-dashboardblockallcontentserviceplugin.js

```diff
@@ -0,0 +1,310 @@
+ */
+YUI.add('ez-dashboardblockallcontentserviceplugin', function (Y) {
+    'use strict';
+
+    /**
+     * Provides the all content dashboard block service plugin class
+     *
+     * @module ez-dashboardblockallcontentserviceplugin
+     */
+    Y.namespace('eZ.Plugin');
+
+    var DEFAULT_ERROR_IDENTIFIER = 'ez-dashboardblockallcontentserviceplugin-error',
+        DEFAULT_ERROR_MESSAGE = 'DashboardBlockAllContentService: error';
+
+    /**
+     * Provides the all content dashboard block service plugin
+     *
+     * @namespace eZ.Plugin
+     * @class DashboardBlockAllContentService
+     * @constructor
+     * @extends eZ.Plugin.ViewServiceBase
+     */
+    Y.eZ.Plugin.DashboardBlockAllContentService = Y.Base.create('dashboardBlockAllContentServicePlugin', Y.eZ.Plugin.ViewServiceBase, [], {
+        initializer: function () {
+            this.onHostEvent('*:getAllContent', this._getAllContent, this);
+            this.onHostEvent('*:gotoContentEdit', this._goToContentEdit, this);
+            this.onHostEvent('*:gotoContentPreview', this._goToContentPreview, this);
+        },
+
+        /**
+         * Gets all content for the all content dashboard block view
+         *
+         * @method _getAllContent
+         * @protected
+         * @param event {Object} event facade
+         */
+        _getAllContent: function (event) {
+            this._loadRootLocation()
+                .then(Y.bind(this._loadRootLocationModelData, this))
+                .then(Y.bind(this._getContentList, this, event.target))
+                .catch(Y.bind(this._handleError, this));
+        },
+
+        /**
+         * Gets a list of content
+         *
+         * @method _getContentList
+         * @protected
+         * @param target {eZ.DashboardBlockAllContentView} a view requesting a content
+         */
+        _getContentList: function (target) {
+            this.get('host').fire('contentSearch', {
+                viewName: 'all-content',
+                loadContentType: true,
+                search: {
+                    criteria: {
+                        SubtreeCriterion: this.get('rootLocation').get('pathString')
+                    },
+                    sortClauses: {
+                        DateModifiedClause: 'DESC'
+                    },
+                    limit: 15
+                },
+                callback: Y.bind(this._setContentList, this, target)
+            });
+        },
+
+        /**
+         * Gets a list of content
+         *
+         * @method _setContentList
+         * @protected
+         * @param target {eZ.DashboardBlockAllContentView} a view requesting a content
+         * @param error {false|Error} error
+         * @param result {Array} array of struct objects containing a `content` property
+         */
+        _setContentList: function (target, error, result) {
+            if (error) {
+                this._handleError({
```

Comment:
better set the `loadingError` attribute provided by AsynchronousView. That way, the retry button (you added it in the template in https://github.com/ezsystems/PlatformUIBundle/pull/589/files#diff-4f07963fcef784ab36aeb705d5588b1eR10) can work.